### ISSUE TITLE
Generic Compressed Data Format

### ISSUE THREAD

**Issue by oeed** [oeed/CraftOS-Standards#21]
It's be good to standardise a format to essentially ZIP something. It would also be good if the format would allow the package to be self-extractable and loadable directly as a table without extracting.

I've had various renditions of this in the past, mainly using textutils.serialise, but it'd be good to have a compressed binary based version if that's remotely feasible.

**Comment by demhydraz** [comment]
This is actually called archiving.

**Comment by oeed** [comment]
Well if someone feels like making a ZIP library for CC we'd all be very happy.

**Comment by demhydraz** [comment]
Oh yes, sure.

**Comment by viluon** [comment]
eeh, no offence, but this is awfully generic, and doesn't actually follow what we tell users to do in the README (and the in-progress `tutorial` branch)! First write the proposal and discuss how the result should look like, then deal with Issues and PRs. This isn't a format that already exists, and there's no need to ~~add~~ invent it at this point. That's just not the purpose of this repo. Develop the format, then make a PR, that's the strategy.

**Comment by oeed** [comment]
Fair point, although I do think having a label for things that need standardising would be useful so people know to make an RFC for it.

**Comment by ghost** [comment]
i think archive and compressing formats should be seperated (see .tar.gz for example), that would also allow someone to only compress one file and dont have to use an archive

**Comment by viluon** [comment]
*Bump*, any reason to keep this around?

**Comment by lyqyd** [comment]
I think this could be closed.
